parts/configure-swarmmode-cluster.sh

```diff
@@ -154,7 +164,7 @@ updateDockerDaemonOptions()
     # also have it bind to the unix socket at /var/run/docker.sock
     sudo bash -c 'echo "[Service]
     ExecStart=
-    ExecStart=/usr/bin/docker daemon -H tcp://0.0.0.0:2375 -H unix:///var/run/docker.sock
+    ExecStart=/usr/bin/docker daemon -H tcp://0.0.0.0:2375 -H unix:///var/run/docker.sock -H fd:// --experimental=true --metrics-addr=0.0.0.0:4999

```

Comment:
This enables experimental mode for everyone even if they don't want to enable it.

This is how I recommend enabling experimental mode and then reading OrchestratorOption during template generate and passing the right flags/settings.

type OrchestratorProfile struct {
        OrchestratorType OrchestratorType json:"orchestratorType"
        KubernetesConfig KubernetesConfig json:"kubernetesConfig,omitempty" 
        // New property to be added to enable SwarmMode experimental
        SwarmModeConfig SwarmModeConfig json:"swarmModeConfig,omitempty"  
}

// New type to be added to enable SwarmMode experimental
type SwarmModeConfig struct {                                                                                
            orchestratorOption[] OrchestratorOptions json:"orchestratorOptions,omitempty"
}

// New type to be added to enable SwarmMode experimental
type OrchestratorOption struct {                                                                                
            Option string
            Value   string
}